Automattic/sensei
# Add default course pattern

Part of #5245
Based on https://github.com/Automattic/sensei/pull/5232

### Changes proposed in this Pull Request

* It removes the Contact Teacher block from the Course CPT template.
* It adds a new pattern with the default course content, and now Course CPT template uses this pattern.
  * It also adds a polyfill to the Pattern block to make sure it will also work in WP 5.8.

### TODO in future PRs

* [ ] Use pattern as default template to lesson.
* [ ] Fix the Course Outline styles in the thumbnail.

### Testing instructions


* Create a new course, add a title and a description in the wizard, and close the modal.
* Make sure the default course template continues working.
* Create a new course, and navigate to the last step.
* Make sure you see a default course pattern.
* Choose it, and make sure it works properly.
* Repeat the tests to make sure it works in WP 5.8 and 6.0, with and without the Gutenberg plugin activated.
